### DISCUSSION TITLE
How to remove "provisioned permission"?

### DISCUSSION THREAD

**Discussion by rjackson** [grafana/grafana#52903] [Dashboards, Panels & Visualizations]
Hi folks,

I have my Grafana instance publicly available and I remove certain dashboards from public-access by removing the "viewer" account's permissions on that dashboard.

I've upgraded to the latest version of Grafana a few days ago, and now I cannot remove "viewer" permissions anymore. There is an icon indicating that the permission is locked, which states "Provisioned permission" when I hover over it:

![Screenshot 2022-07-27 at 17 44 43](https://user-images.githubusercontent.com/602850/181303926-249b07fb-a423-43ea-91ab-c7b5da43e013.png)

Can anybody advise how I can revoke the permissions granted to "viewer" when it appears to be a "provisioned permission"?

Thanks!

**Comment by exander77** [comment]
We have the same issue.

**Comment by gabrielsborini** [comment]
Olá, provavelmente estes seus gráficos estão dentro de uma pasta, e é nela que precisa alterar as permissões.
- Coloque o mouse sobre a pasta e clique em Go to folder.
- Permissions.
- Altere as permissões da pasta conforme o necessário.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
